.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           command: |
             mkdir ./results
             pylint playback > ./results/pylint.xml
-            flake8 playback --max-line-length=120 --format junit-xml > ./results/flake8.xml
+            flake8 tests playback --max-line-length=120 --format junit-xml > ./results/flake8.xml
```

Comment:
what about examples dir?

Reply:
flake yes, pylint there are many issues, I don't have time now to deal with them.